tests/Integration/Config/configurations/anchors.yaml

```diff
@@ -3,7 +3,7 @@
 file_format: "0.3"
 exporters:
     otlp: &otlp-exporter
-        protocol: http/protobuf
+        encoding: protobuf
```

Comment:
We should remove this file, `endpoint` must include the signal path.

Reply:
moved endpoint out of the anchor. I assume this is still a worthwhile test that anchors work...